Automattic/sensei
# Fix lesson quick edit and bulk edit for quiz settings

Fixes #3382 #4117 

### Changes proposed in this Pull Request

* Add 3 post meta fields to bulk and quick edit fields in the lesson admin page: `show_questions`, `random_question_order`, `quiz_grade_type`
* Save persist these post meta fields to the database when save or update button is clicked.

### Testing instructions

* Create a lesson.
* On the Lessons page, hover over the lesson and click Quick Edit.
* Set a value for Pass required, Pass Percentage and Enable quiz reset button.
* Click Update.
* Select the lesson and view the quiz settings.
* You should see the updated settings
